### PR TITLE
doc examples in FAQ for HTML entities and mounting Node[Seq]

### PR DESCRIPTION
**Pull request by bbarker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -319,6 +319,35 @@ Elements:
...
+}
+```
+
+#### How do I use HTML entities?
+
+There are a couple of ways. The first is to inject it directly into a node. To get two non-breaking spaces, you might do:
+`<span>&npsp;&npsp;</span>`. More generally, you will need `scala.xml.EntityRef` to reference an entity directly in Scala:
+
+```scala
+def space(nn: Int): Node = Group((0 until nn).map{_ => EntityRef("nbsp")})
```
Comment (OlivierBlanvillain) [inline README.md]:
```scala
Seq.fill(nn)(EntityRef("nbsp"))
```

File: README.md [hunk 2]
```diff
@@ -319,6 +319,35 @@ Elements:
...
+<span>{space(2)}</span> // results in  <span>"&npsp;""&npsp;"</span>
+
```
Comment (OlivierBlanvillain) [inline README.md]: Extra new line

File: README.md [hunk 3]
```diff
@@ -319,6 +319,35 @@ Elements:
...
+```
+
+#### How do I use HTML entities?
+
+There are a couple of ways. The first is to inject it directly into a node. To get two non-breaking spaces, you might do:
+`<span>&npsp;&npsp;</span>`. More generally, you will need `scala.xml.EntityRef` to reference an entity directly in Scala:
+
+```scala
+def space(nn: Int): Node = Group((0 until nn).map{_ => EntityRef("nbsp")})
+<span>{space(2)}</span> // results in  <span>"&npsp;""&npsp;"</span>
```
Comment (OlivierBlanvillain) [inline README.md]: Why are there quotes (`"`) here?

Comment (bbarker) [inline README.md]: That's how it shows up in the DOM inspector; here's an example from `<h1 style = "color: #FFFFFF">{space(3)}{ced2ar}</h1>`:
image

Comment (bbarker) [inline README.md]: Maybe a scala.xml thing?

**Comment by bbarker** [comment]
Added the test, good way to get warmed up with testing mhtml!

File: README.md (hunk 3 above)
Comment (OlivierBlanvillain) [inline README.md]: Hmm that's weird, but whatever if the browser renders things expected.

**Comment by OlivierBlanvillain** [comment]
Many thanks, merging :-)

**Pull request merged**